auth-web/src/views/auth/staff/SafeEmailView.vue

```diff
@@ -44,28 +60,52 @@ export default defineComponent({
   name: 'SafeEmailView',
   setup () {
     const safeEmails = ref<SafeEmail[]>()
+    const alertMessage = ref('')
```

Comment:
would wrap these in toRefs / computed, look at `state` for some of the other components.. 

Returning all these refs are easy to miss